src/components/card.js

```diff
@@ -11,7 +11,7 @@ const sizes = Object.freeze({
 })
 
 const Card = ({ classes, size, ...rest }) => {
-  const _classes = { ...classes, root: `card__root-container ${['rounded-sm', sizes[size], classes.root].join(' ')}` }
+  const _classes = { ...classes, root: `card__root-container ${[classes.root, 'rounded-sm', sizes[size]].join(' ')}` }
   return (
     <CardBase classes={_classes} {...rest} />
   )
```

Comment:
lets add a classname for header, content & footer like root card__header-container, and so on

Reply:
Should I add these classes' names to the Card component or the CardBase component?

Reply:
@amy83762100 card component passing it down to cardBase component thourgh classes prop